sysutils/salt/Portfile

```diff
@@ -23,56 +23,95 @@ if {$subport eq $name} {
     github.setup            saltstack salt ${version} v
     revision                1
 
-    python.versions         27 35 36
+    python.versions         35 36 37
     categories              sysutils python
 
-    checksums       rmd160  47b77490e40fdc2ec4c5d262003afecdb0b8e94a \
-                    sha256  0a65913142577a70924a283a99dc67612630a9601fd5c83d660bad74b84357d4 \
-                    size    14987367
+    checksums rmd160 f1c2a4f0609c3cd64f3b98c3a3bf8efe39703c75 \
+              sha256 c5c16a328b7971fecf1dc8902f2315387894613b198581c090063abc67df8fb7 \
+              size   15255885
 
     notes    "Salt startupitems are installed by subports salt-minion, salt-master, salt-syndic, salt-api."
 
-    if {![variant_isset python35] && ![variant_isset python36]} {
-        default_variants +python27
+    if {![variant_isset python36] && ![variant_isset python37]} {
+        default_variants +python36
```

Comment:
this doesn't look right; you likely want to have the lastest python version as the default so it should probably be:
```
if {![variant_isset python35] && ![variant_isset python36]} {
        default_variants +python37
```

additionally, is Python 3.8 also supported; if so please add it.